examples/bank/bank.go

```diff
@@ -117,9 +117,6 @@ func (bank *Bank) continuousMoneyTransfer(cash int64) {
 			if err := runner.Run(batchRead); err != nil {
 				return err
 			}
-			if batchRead.Results[0].Err != nil {
```

Comment:
What if you want to know which operation in a batch caused an error? Not sure why you'd want to know that, but that's the use case for `Result.Err`.